src/style.css

```diff
@@ -0,0 +1,104 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 14px 20px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  margin-top: 45px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__review {
+  padding-top: 15px;
+  padding-bottom: 21px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__sku {
+  margin-top: 5px;
+}
+
+.card__rate {
+  margin-top: 15px;
+}
+
+.card__price-area {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__price {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__button {
+  margin-top: 21px;
+  display: block;
+  background-color: #00acdc;
+  text-align: center;
+  line-height: 40px;
+  height: 40px;
+  width: 170px;
+  text-transform: uppercase;
+  border-radius: 5px;
+  font-weight: bold;
+  font-size: 14px;
+  color: white;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: white;
+  color: #00acdc;
+}
+
+.rate__star {
+  display: inline-block;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.rate {
+  display: flex;
+}
+
+.stars--1 span.rate__star:nth-child(-n+1),
+.stars--2 span.rate__star:nth-child(-n+2),
+.stars--3 span.rate__star:nth-child(-n+3),
+.stars--4 span.rate__star:nth-child(-n+4),
+.stars--5 span.rate__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
can't see none of this classes but stars--4

Reply:
Removed unnecessary classes